next.config.js

```diff
@@ -8,6 +8,9 @@ const nextConfig = {
 
         return config
     },
+    env: {
```

Comment:
nit;

Feels unnecessary to have this include the name Django in its name here. It's just the base url for the backend, we could make this not be a Django server later and that decision should be irrelevant to the frontend, as long as the base url is provided.